tests/foreman/virtwho/cli/test_esx.py

```diff
@@ -420,3 +421,53 @@ def test_positive_foreman_packages_protection(
         assert virt_who_instance == 'OK'
         target_sat.cli.VirtWhoConfig.delete({'name': virtwho_config['name']})
         assert not target_sat.cli.VirtWhoConfig.exists(search=('name', form_data['name']))
+
+    @pytest.mark.tier2
+    def test_positive_deploy_configure_hypervisor_password_with_special_characters(
+        self, default_org, form_data, target_sat
+    ):
+        """Verify " hammer virt-who-config deploy hypervisor with special characters"
+
+        :id: 9892a94e-ff4b-44dd-87eb-1289d4a965be
+
+        :expectedresults: Config can be created and deployed with any error
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+
+        :BZ: 1870816,1959136
```

Comment:
These BZ's have customer cases attached, please include `:customerscenario: true` in the metadata.